tensorboard/data/server/run.rs

```diff
@@ -104,7 +104,7 @@ impl StageTimeSeries {
     fn commit(&mut self, tag: &Tag, run: &mut commit::RunData) {
         use pb::DataClass;
         match self.data_class {
-            DataClass::Scalar => self.commit_to(tag, &mut run.scalars, EventValue::into_scalar),
+            DataClass::Scalar => self.commit_to(tag, &mut run.scalars, |ev, _| ev.into_scalar()),
```

Comment:
I'm not sure I understand why this had to change.  Are we expecting to modify this enrich function, in some way that doesn't also apply to L120's `EventValue::into_blob_sequence`?

Reply:
It changes because of this line of the diff:

```diff
diff --git a/tensorboard/data/server/run.rs b/tensorboard/data/server/run.rs
--- a/tensorboard/data/server/run.rs
+++ b/tensorboard/data/server/run.rs
@@ -125,7 +125,7 @@ impl StageTimeSeries {
 
     /// Helper for `commit`: writes staged data for this time series into storage for a statically
     /// known data class.
-    fn commit_to<V, F: FnMut(EventValue) -> Result<V, commit::DataLoss>>(
+    fn commit_to<V, F: FnMut(EventValue, &pb::SummaryMetadata) -> Result<V, commit::DataLoss>>(
         &mut self,
         tag: &Tag,
         store: &mut commit::TagStore<V>,
```

The callback now takes `(EventValue, &pb::SummaryMetadata)` rather than
just `EventValue`. But `EventValue::into_scalar` is still a function of
only an `EventValue`. So this lambda drops the unused parameter.

Line 120 doesn’t change because `EventValue::into_blob_sequence` used to
take `EventValue` and now takes `(EventValue, &pb::SummaryMetadata)`,
just like the callback, so it typechecks both before and after.

Reply:
Ah, thanks.  Yeah, both commit_to and into_* have changed, it makes sense